wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md

```diff
@@ -60,7 +63,7 @@ How long the text stays on the screen, including the fade in and out.
 Font size in points.
```

Comment:
add this link to explain what points are `[points](https://en.wikipedia.org/wiki/Point_(typography))`

tl;dr its the smallest unit of measure that is used for the height of a font (it equates to 1/72 of an inch, but converting this to pixels will vary as screen resolutions follows)